Tests/babel-plugin-replace-bigint.js

```diff
@@ -0,0 +1,13 @@
+module.exports = function() {
```

Comment:
Maybe some comments about why this is needed.

Reply:
Fixed